composer.json

```diff
@@ -4,11 +4,12 @@
     "type": "cakephp-plugin",
     "license": "MIT",
     "require": {
-        "php": ">5.6",
-        "cakephp/cakephp": "^3.7"
+        "php": ">=7.2",
+        "cakephp/cakephp": "^4.0.1"
     },
     "require-dev": {
         "cakephp/cakephp-codesniffer": "^3.0",
+        "phpunit/phpunit": "^8.0.4",
```

Comment:
Don't include PHPUnit in the composer.json as its installed with the CI commands. Will add phpunit setup scripts for testing locally.

Reply:
Will remove.